check_process

```diff
@@ -1,25 +1,30 @@
 ;; Test complet
 	; Manifest
-		domain="sub.domain.tld"	(DOMAIN)
-		path="/"	(PATH)
+		domain="domain.tld"
+		path="/path"
+		is_public=1
+		language="fr"
+		admin="john"
+		password="1Strong-Password"
+		port="666"
 		backend="domain.tld"
-		is_public=1 (true|false)
 	; Checks
 		pkg_linter=1
-		setup_sub_dir=0
+		setup_sub_dir=1
```

Comment:
```suggestion
		setup_sub_dir=0
```

So it looks like this app can't be installed in a subpath, cf the `path_url="/"` in the install script